### ISSUE TITLE
Add Link to Source Code of DPGs

### ISSUE THREAD

**Issue by seanmarcia** [DPGAlliance/publicgoods-scripts#129]
# Summary

Currently, people who come to the Digital Public Goods (DPGs) registry don't have an easy way to view the code of the repositories. This forces them to visit the project, search for a link to GitHub (or other hosting platform) and then navigate there. For some DPGs this is a frustrating experience as this is something available. I'd like to add a link in the registry to the code.

# Justification
This will allow technologists and others who are interested in contributing to DPG projects to quickly find the source code and not leave frustrated. Several projects link to production sites and finding the code requires internet searches or more. Some of the key benefits I see of listing this are:
- This will make it easier for potential code contributors to find where to go to make these contributions. 
- It will also make it easier folks who are thinking of adopting the project but want to evaluate the code before doing so.

# Details
It *looks* like this could be accomplished by adding a link in the `ListItem` function in following file: 
`packages/registry/src/index.js` which appears to be generating the registry. 

It would involve editing the sections which set the `name` attribute and specifically it could be accomplished in lines 261-266 of the above file.

# Other considerations

The `json` payload contains a `repositories` section with a link to the repository/source code of the project. It should be noted that a few projects list multiple repositories so I'd propose taking the first repository and using it. 

# Criteria for completion
- [ ] Add a `Source Code` link in the name section of the registry that links to the source code if it exists.
- [ ] When there are multiple repositories available link to the first repository.

**Comment by nathanfletcher** [comment]
@prajectory & @nathanbaleeta  What do you think? Since I'm also working on another issue on the registry page

**Comment by seanmarcia** [comment]
Hey Nathan, I'm happy to work on this. I'll shortly be submitting a decently sized PR that is a refactor of the registry page that should make future contributions .

Basically, it breaks the page into several components.

I did a small mock up of how I've broken things up here: 
![GithubUNV3](https://user-images.githubusercontent.com/667909/190547994-9134a1a9-dfef-4c0b-a778-93ce9785fa62.png)

Basically, I've taken all the logic in the main page and broken it out into two components. <Filter/> and <List/> with filter having 4 sub components and List having <ListItem> under it and it made up of 4 components. I kept List and ListItem to reflect the current naming conventions but plan on changing them to <RegistryList/> and <RegistryItem/> before submitting.
